styles.css

```diff
@@ -1,21 +1,80 @@
 
+.topnav {
+  overflow: hidden;
+  background-color: gray;
+  position: relative;
+}
```

Comment:
- [ ] Please know that as the previous reviewer mentioned, the color of the navbar shouldn't be gray, it should be the same one that is mentioned in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27), kindly change that to the correct one.

Hint: Please go to the Figma design, click on the toolbar then on the right side of the page you'll find an inspect panel where you can check the CSS used to make the design, check this picture it should make it clear for you.

![image](https://user-images.githubusercontent.com/20567503/176714007-4a726b1a-16b8-42d3-87cf-c4848704dd46.png)